backend/testClient.spec.js

```diff
@@ -0,0 +1,175 @@
+const { createTestClient } = require('apollo-server-testing');
+
+const { typeDefs } = require("./typeDefs");
+const { resolvers } = require("./resolvers");
+
+const { ApolloServer, gql } = require('apollo-server');
+
+const { jwt } = require('jsonwebtoken');
+
+let token;
+
+describe("ServerTest", () => { 
+
+  const server = new ApolloServer({
+    typeDefs,
+    resolvers
+  });
+
+  const { query, mutate } = createTestClient(server);
+
+  it('test Login', async () => {
```

Comment:
"it test login"? :thinking: try to use simple-present and avoid the word "should":

```js
describe('mutation', () => {
  describe('login', () => {
    it('returns a JWT token', () => {
      // ...
    })
  })
})
```

The 'it` should read like a nice English sentence: "It behaves in some way". Consider all your test case descriptions as the documentation/specification of your code.